run_dir/design/flowcell.html

```diff
@@ -92,6 +92,16 @@
                       {% elif run_setup < 100 %}
                         {% set q30_threshold = 80.0 %}
                       {% end %}
+                    {% else %}
+                      {% if run_setup >= 250 %}
+                        {% set q30_threshold = 60.0 %}
+                      {% elif run_setup >= 150 %}
+                        {% set q30_threshold = 75.0 %}
+                      {% elif run_setup >= 100 %}
+                        {% set q30_threshold = 80.0 %}
+                      {% elif run_setup < 100 %}
+                        {% set q30_threshold = 85.0 %}
+                      {% end %}
```

Comment:
Which result do you want if run_setup is not defined ? You could change the default to get that behaviour